src/content/configuration/index.mdx

```diff
@@ -0,0 +1,455 @@
+---
```

Comment:
i wonder why this shows as a diff and not as a move?

Reply:
GitHub says the file was removed, I don't remember what I did exactly